### PR TITLE
Vertex Web surf added

### PR DESCRIPTION
**Pull request by anishaxtha**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -721,6 +721,22 @@
         "youtube": null
       }
     },
+    {
+      "name": "Vertex Web Surf Pvt. Ltd.",
+      "establishedAt":1363198500,
```
Comment (susanhsrestha) [inline data/companies.json:726]: A little spacing between the key-value pair is needed to preserve the format. Also, I don't think the timestamp is correct.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

**Comment by susanhsrestha** [comment]
You can refer to the solutions provided in #70 to get the updated established date of the company.

**Comment by anishaxtha** [comment]
sure! I will change it

**Comment by anishaxtha** [comment]
hello maintainers, I have made the changes, so could u please review and merge it Thank you:)

**Review by susanhsrestha — APPROVED** [review]
LGTM! Thank you for your contribution @anishaxtha

File: data/companies.json [hunk 2]
```diff
@@ -721,6 +721,22 @@
         "youtube": null
       }
     },
+    {
+      "name": "Vertex Web Surf Pvt Ltd.",
+      "establishedAt": 1453832100,
+      "location": "Samakhusi, Kathmandu",
+      "website": "https://www.vertexwebsurf.com.np/",
+      "email": "info@vertexwebsurf.com.np",
+      "phone": "01-4388511",
```
Comment (thenishantsapkota) [inline data/companies.json:730]: The format for the phone number is `+977-14388511`

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please properly format code using prettier and push the changes again.

**Comment by thenishantsapkota** [comment]
@susanhsrestha I am once again informing you that we should properly review the PRs and not approve them without meeting the mininum criteria required.

File: data/companies.json (hunk 2 above)
Comment (susanhsrestha) [reply data/companies.json:730]: > The format for the phone number is `+977-14388511`

I checked but the some previously merged codes don't follow this formatting for phone number. So, I let it be.

**Pull request closed without merging**